modules/tinymce/tools/modules/grunt-webpack.js

```diff
@@ -29,6 +29,11 @@ let create = (entries, tsConfig, outDir, filename) => {
     },
     module: {
       rules: [
+        {
+          test: /\.(jpe?g|png|gif|svg)$/i,
```

Comment:
```suggestion
          test: /\.(svg)$/i,
```
We definitely don't want regular images using the raw-loader, so just dropping it back to what we need for now.

Reply:
ah yeah this was added before the swap to `raw`, thanks